xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-ui/src/main/resources/XWiki/OfficeImporterTranslations.xml

```diff
@@ -40,7 +40,7 @@
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
   <content>office.createForm.type.name=Page from Office
-office.createForm.type.description=Import office documents into wiki pages
+office.createForm.type.description=Import Office documents into wiki pages
```

Comment:
It looks like a good idea to use uppercase `Office`, since it is usually a shorter version of saying `Microsoft Office` or `Open Office` or `Libre Office`, etc., in call cases the work being part of a name.

However, you need to be careful in preserving consistency. Looking at the translations below, you did not update them as well. If you make this change, you need to look a the other places where it is being used and adapt them accordingly.

Reply:
Ok. Changed the other phrases